### DISCUSSION TITLE
Da li ugovor u servisu može da prati realizaciju kao okvrina prodjana porudžbina?

### DISCUSSION THREAD

**Discussion by Bmiljan** [NPSBeograd/NPS-Support#82] [Q&A] [answered]
Da li ugovor u servisu može da prati realizaciju kao okvrina prodjana porudžbina?

**Comment by IvanaBasara** [comment, marked as answer]
Postoji akcija da sa ugovora o servisiranju pokrenete izveštaj (Putanja: Izveštaj - Fakturisanje ugovora), na ovom izveštaju će te imati podatke za koji je period kreirana faktura i vremenski period za buduće fakture, kao i iznose.
Takođe sa ugovora o servisiranju možete da vidite koje su sve fakture proknjižene za ugovor.
Putanja: Povezano – Pregled – Pregled servisa – Proknjižene servisne fakture
